### PR TITLE
DEV-172 fix building

### PR DESCRIPTION
**Pull request by taybenlor**
@wryun have a squiz at this

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
+set -ue -o pipefail -o errtrace
+
 DEPLOY_PATH=${DEPLOY_HASH:-local}
 
-pxt staticpkg -r "$DEPLOY_PATH"
+node node_modules/pxt-core/built/pxt.js staticpkg -r "$DEPLOY_HASH"
```
Comment (wryun) [inline build.sh]: Should be DEPLOY_PATH?

**Review by wryun — CHANGES_REQUESTED** [review]

File: build.sh [hunk 2]
```diff
@@ -1,6 +1,8 @@
+set -ue -o pipefail -o errtrace
```
Comment (wryun) [inline build.sh:2]: also good to add #!/bin/bash at the top and make it executable.

File: build.sh (hunk 1 above)
Comment (taybenlor) [reply build.sh]: Whoops :P copy paste fail

**Comment by taybenlor** [comment]
@wryun fixed that up, it was already `+x` is there anything else I need to do to make it executable?

**Comment by wryun** [comment]
Sorry, missed the +x. Looks good!

**Review by wryun — APPROVED** [review]
LGTM.

**Pull request merged**